packages/transmute-framework/src/stream-model/__tests__/applyEvents.spec.js

```diff
@@ -29,7 +29,7 @@ describe('applyEvents', () => {
     eventStore = await eventStore.clone(accounts[0]);
   });
 
-  it('idempotent', async () => {
+  it('should be idempotent', async () => {
```

Comment:
Maybe I'm missing something, but how can applyEvents not be idempotent with such a reducer?

Reply:
yep, this reducer is a poor choice.